src/utils/authentication/AuthenticationManager.ts

```diff
@@ -0,0 +1,18 @@
+import firebase from 'firebaseSetup';
+
+export const AuthenticationManager = {
+    signInAnonymously,
+};
+
+function updateProfile(displayName?: string, photoURL?: string) {
+    firebase.auth().currentUser?.updateProfile({
+        displayName,
+        photoURL
+    });
+}
+
+function signInAnonymously(displayName?: string, photoURL?: string) {
+    return firebase.auth().signInAnonymously().then(_ => {
+        updateProfile(displayName, photoURL);
```

Comment:
The `updateProfile` call might also throw an error which is uncaught / handled here. Do you intend to handle it somewhere else?

Reply:
Yes it's very important to handle errors so the user knows if something went wrong. I just wasn't sure what a good solution would be. What do you think about Toasts? @bitionaire We could display a toast with an error message for example if the profile couldn't be updated.